data/assets/02_reset.css

```diff
@@ -1,3 +1,5 @@
+@import './01.css';
```

Comment:
I don't see much reason for the "reset" CSS to import the global config variables, because it is unlikely it will need to refer to any of them. I'm actually more questioning the need for the "reset". Why do we actually need the reset?